redshift_sqlalchemy/dialect.py

```diff
@@ -264,6 +527,167 @@ def create_connect_args(self, *args, **kwargs):
         default_args.update(cparams)
         return cargs, default_args
 
+    @util.memoized_property
+    def _isolation_lookup(self):
+        extensions = __import__('psycopg2.extensions').extensions
```

Comment:
Tests all pass with this method removed. Looking at history, it dates back to 2013 in this Redshift dialect, and there have been significant changes to the same method in the psycopg2 dialect since then.

I'm in favor of removal.